python-sdk/src/astro/session.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from airflow.exceptions import AirflowException
+from sqlalchemy.sql import ClauseElement
+
+
+class Session:
```

Comment:
@dimberman I think the name is a little generic, can we change it to something that indicates the SQL link?